src/content/docs/apm/agents/nodejs-agent/installation-configuration/install-nodejs-agent-docker.mdx

```diff
@@ -57,19 +62,25 @@ With just a few additions your existing Dockerfile can be used with our Node.js
   Don't include your license key in your Dockerfile or Docker image. For more information, see [our documentation on license key security](/docs/accounts/install-new-relic/account-setup/license-key#license-key-security).
 </Callout>
 
-In addition to setting your application name or license key, you can set [other configuration options](/docs/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration) by starting your container with the `-e` option. For example, to enable [distributed tracing](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing), use:
+<Callout variant="tip">
+  If your Node.js agent is older than [v8.3.0](/docs/release-notes/agent-release-notes/nodejs-release-notes/node-agent-8-3-0) and you want to use [distributed tracing](/docs/understand-dependencies/distributed-tracing/get-started/introduction-distributed-tracing), you'll need to enable it by setting the environment variable `NEW_RELIC_DISTRIBUTED_TRACING_ENABLED` to `true`.
```

Comment:
These call outs about versioning makes me believe there should be a compatibility section with all of this content grouped together AHEAD of procedure. Having it dispersed throughout the doc can be confusing and break flow, and then customers can't go to one section for reference about versioning. Can we make these version callouts h2s with everything lumped together?